lib/parse-args.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const minimist = require('minimist');
+
+const parse = (args) => {
+  if (!args || args.length === 0) {
+    throw new Error('No src folder or pattern specified');
```

Comment:
I prefer messages that tell the user how to resolve, maybe "Specify a src folder or pattern"